iModelCore/Units/PublicAPI/Formatting/FormattingApi.h

```diff
@@ -294,6 +294,10 @@ struct NumericFormatSpec
     void SetCounterClockwiseAngle(bool setTo) { SetTraitsBit(FormatTraits::CounterClockwiseAngle, setTo);}
     bool IsCounterClockwiseAngle() const {return GetTraitBit(FormatTraits::CounterClockwiseAngle);}
 
+    // TODO - Naron: this should be somewhere else?

```

Comment:
It's an alright place, because the other numeric formatting code is in this class, too. I think it's fine. Another possible location for this would be in format directly (alongside formatQuantity)